README.md

```diff
@@ -1,70 +1,101 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid19 Metrics
```

Comment:
- [ ] The content of the video presentation of the project should include:
  - a description of the project.
  - a demo of the project features.
  - you should also highlight some interesting piece of code or something you built that you are very proud of.

The current video doesn't contain these, kindly record another one.